content/blog/2025-anndata-012.md

```diff
@@ -0,0 +1,74 @@
++++
+title = "Improved i/o in anndata 0.12"
+date = 2025-07-25T00:00:05+01:00
+description = "With zarr v3 support and xarray integration, anndata takes a big step forward when it comes to i/o tasks."
+author = "Ilan Gold"
+draft = false
++++
+
+# 0.12 released
+
+We're happy to announce that `anndata` 0.12 is out now!
+Check out [the changelog](https://anndata.readthedocs.io/en/stable/release-notes/index.html#v0-12-0) for a full list of changes.
+Here, we want to give our users a bit of a deep dive into the new functionality.
+We have lots of great features, like zarr v3 support (package and format), full lazy loading, and new API customisability! Let’s dive in!
+
+## Zarr v3
+
+[Zarr v3 as a file format](https://zarr-specs.readthedocs.io/en/latest/v3/core/index.html) provides improved cloud support, support for sharding to reduce the number of files created in a zarr store, and improved support for extensibility.
+Check out this graphic from the zarr docs.
```

Comment:
This sentence in the middle of the paragraph sounds a little weird, maybe this makes it clearer that the second half of the paragraph explains the graphic?

```suggestion
Check out the following graphic from the zarr docs:
```